Source/WebCore/Modules/geolocation/GeoNotifier.cpp

```diff
@@ -69,14 +69,14 @@ bool GeoNotifier::hasZeroTimeout() const
     return !m_options.timeout;
 }
 
-void GeoNotifier::runSuccessCallback(GeolocationPosition* position)
+void GeoNotifier::runSuccessCallback(GeolocationPosition& position)
 {
     // If we are here and the Geolocation permission is not approved, something has
     // gone horribly wrong.
     if (!m_geolocation->isAllowed())
         CRASH();
 
-    protectedSuccessCallback()->handleEvent(position);
+    protectedSuccessCallback()->handleEvent(&position);
```

Comment:
You probably want to address this FIXME: https://searchfox.org/wubkat/rev/c0417a386c3062049eb1abec31004d7ecde8a4d1/Source/WebCore/Modules/geolocation/PositionCallback.idl#29

and pass `position` directly, instead of `&position`